### PR TITLE
Manifest diff prints

### PR DESCRIPTION
**Pull request by lasote**
Changelog: Feature: Increase debugging information when an error uploading a recipe with different timestamp occurs.

### CODE REVIEW

File: conans/client/cmd/uploader.py [hunk 1]
```diff
@@ -158,8 +158,31 @@ def _check_recipe_date(self, conan_ref, remote):
...
+            self._user_io.out.info("Remote manifest:")
+            self._user_io.out.info(remote_recipe_manifest)
+            self._user_io.out.info("Local manifest:")
+            self._user_io.out.info(local_manifest)
+            difference = remote_recipe_manifest.difference(local_manifest)
+            if "conanfile.py" in difference:
+                contents = tools.load(os.path.join(self._client_cache.export(conan_ref),
+                                                   "conanfile.py"))
+                endlines = "\\r\\n" if "\r\n" in contents else "\\n"
+                self._user_io.out.info("Local 'conanfile.py' using '%s' line-ends" % endlines)
```
Comment (jgsogo) [inline conans/client/cmd/uploader.py]: There can be mixed line endings in the same file (Visual Studio people knows about it, it is a common error 😩 )

File: conans/client/cmd/uploader.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
+import os
 import time
-
-from conans import load
+from conans import tools
```
Comment (jgsogo) [inline conans/client/cmd/uploader.py]:
```suggestion
from conans.util.files import load
```

Get rid of _globals_ polluting the code base.

File: conans/client/cmd/uploader.py [hunk 3]
```diff
@@ -158,8 +158,31 @@ def _check_recipe_date(self, conan_ref, remote):
...
+    def _print_manifest_information(self, remote_recipe_manifest, local_manifest, conan_ref, remote):
+        try:
+            self._user_io.out.info("\n%s" % ("-"*40))
+            self._user_io.out.info("Remote manifest:")
+            self._user_io.out.info(remote_recipe_manifest)
+            self._user_io.out.info("Local manifest:")
+            self._user_io.out.info(local_manifest)
+            difference = remote_recipe_manifest.difference(local_manifest)
+            if "conanfile.py" in difference:
+                contents = tools.load(os.path.join(self._client_cache.export(conan_ref),
```
Comment (jgsogo) [inline conans/client/cmd/uploader.py]:
```suggestion
                contents = load(os.path.join(self._client_cache.export(conan_ref),
```

File: conans/client/cmd/uploader.py (hunk 1 above)
Comment (lasote) [reply conans/client/cmd/uploader.py]: Sure. It is only a bit more information that can be totally useless in some cases but very useful sometimes.

**Pull request merged**